mtp_noms_ops/apps/security/tests/test_views.py

```diff
@@ -2677,7 +2678,7 @@ class BaseCheckViewTestCase(SecurityBaseTestCase):
     prisoner_id = 3
     credit_id = 5
 
-    credit_created_date = datetime.datetime.now()
```

Comment:
this was a timezone-naive date and hence usages of `credit_created_date` in mocked api responses could not be parsed properly